katdal/datasources.py

```diff
@@ -291,27 +285,6 @@ def __init__(self, store, chunk_info, corrprods):
                          dtype=darray['flags'].dtype)
         darray['flags'] = flags
```

Comment:
I guess this is not necessary anymore now that the "filled-" dask arrays are gone.

Reply:
Scratch this...